woocommerce/woocommerce-ios
# Product variations: Update Networking and Yosemite to support sorting list

Part of #10619 


## Description

This PR updates the Networking and Yosemite layers with new params for sorting product variations. The change is done following the [API doc](https://woocommerce.github.io/woocommerce-rest-api-docs/#list-all-product-variations) and the current behavior of the product list. We are supporting only sorting by date and title.

## Testing instructions

- Log in to the app and navigate to the Products tab.
- Select an existing variable product or create a new one with variations if needed.
- The variation list should still be listed as before as we are using the default values for `order` and `orderby`. Generating new variations should still work properly too.

## Screenshots

<img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/df622acd-611c-4a1c-8fa5-db109610e56d" width=320 />


---
